payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SimpleDialogElementUI.kt

```diff
@@ -5,26 +5,25 @@ import androidx.compose.material.AlertDialog
 import androidx.compose.material.Text
 import androidx.compose.material.TextButton
 import androidx.compose.runtime.Composable
-import androidx.compose.runtime.MutableState
 import com.stripe.android.uicore.StripeTheme
 import com.stripe.android.uicore.elements.H6Text
 
 @Composable
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 fun SimpleDialogElementUI(
-    openDialog: MutableState<Boolean>,
+    openDialog: Boolean,
```

Comment:
This breaks other places where this composable is used. If we switch from `MutableState<Boolean>` to `Boolean`, we need to make the confirm and dismiss listeners non-nullable to force consumers to handle them.